build/repo.targets

```diff
@@ -83,11 +87,12 @@
 
   <ItemGroup>
 
-    <PhysicalScenarios Include="-n PlaintextPlatform --webHost KestrelLibuv $(PlaintextPlatformJobs)" />
+    <PhysicalScenarios Include="-n PlaintextPlatform --webHost KestrelSockets $(PlaintextPlatformJobs)" />
     <PhysicalScenarios Include="-n Plaintext --webHost KestrelSockets $(PlaintextJobs)" />
     <PhysicalScenarios Include="-n Plaintext --webHost KestrelSockets $(PlaintextJobs)" Description="Baseline" AspNetCoreVersion="Current" RuntimeVersion="Current"/>
     
     <!-- Optimal ThreadCount for current hardware -->
+    <PhysicalScenarios Include="-n PlaintextPlatform --webHost KestrelLibuv --kestrelThreadCount 2 $(PlaintextPlatformJobs)" />
```

Comment:
We'll need to note this change (thread count) as we are already storing results for a week on an automatic count